src/app-init/router.js

```diff
@@ -43,6 +44,7 @@ routerConfig(
       { name: ROUTE_FRAGMENT_ADD, path: '/fragment/add' },
       { name: ROUTE_FRAGMENT_EDIT, path: '/fragment/edit/:fragmentCode' },
       { name: ROUTE_FRAGMENT_DETAIL, path: '/fragment/view/:fragmentCode' },
+      { name: ROUTE_DATAMODEL_LIST, path: '/datamodel' },
```

Comment:
please add underscore: ROUTE_DATA_MODEL_LIST (as we did in ROUTE_DATA_MODEL_ADD )